easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'MEM'
+local_commit = '6b92476'
+version = '20191023'
+versionsuffix = '-R-%(rver)s'
```

Comment:
@boegel I thought this was not necessary, as R-3.6.2 is the default for 2019b?
apparently some easyconfigs have added it, and other haven't...

Reply:
Yeah, we're not exactly consistent there... I'll get rid of it here.